src/xtgeo/grid3d/_grid3d_utils.py

```diff
@@ -81,8 +91,9 @@ def scan_dates(pfile, maxdates=MAXDATES, dataframe=False):
     return zdates
 
 
-def _scan_ecl_keywords(pfile, maxkeys=MAXKEYWORDS, dataframe=False):
-    logger.info("Scanning ECL keywords...")
+def _scan_ecl_keywords(
+    pfile: _XTGeoFile, maxkeys: int = MAXKEYWORDS, dataframe: bool = False
+) -> Union[List[KeywordTuple], pd.DataFrame]:
     cfhandle = pfile.get_cfhandle()
 
     # maxkeys*10 is used for 1D kewords; 10 => max 8 letters in eclipse +
```

Comment:
Is an (old) typy here "kewords" --> "keywords"

Reply:
Fixed, thanks